lib/Client.js

```diff
@@ -2064,6 +2064,21 @@ class Client extends EventEmitter {
         }));
     }
 
+    _formatImage(url, format, size) {
+        if(!format || !Constants.ImageFormats.includes(format.toLowerCase())) {
+            format = url.includes("/a_") ? "gif": this.options.defaultImageFormat;
+        }
+        if(
+            !size ||
```

Comment:
Bit of an odd if condition style

Reply:
It's this way to remain consistent to what it was before, the only difference having `!size ||`

Reply:
mhm yea totally